packages/protocol/contracts/dapis/SamplePriceDataFeed.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.4;
+
+import "./RrpDapiServer.sol";
+
+// A sample price data feed implemented as a dAPI client
+contract SamplePriceDataFeed {
+    RrpDapiServer public rrpDapiServer;
+    bytes16 public latestDapiId;
+    int256 public latestAnswer;
+
+    constructor (address _rrpDapiServer)
+    {
+        rrpDapiServer = RrpDapiServer(_rrpDapiServer);
+    }
```

Comment:
I believe `registerDapi` should go in here as well.

Reply:
Let's cover that stuff in following PRs